Asterisk.2013/Asterisk.NET/Manager/Event/QueueMemberPausedEvent.cs

```diff
@@ -1,10 +1,13 @@
+using System;
+
 namespace AsterNET.Manager.Event
 {
 	/// <summary>
 	/// A QueueMemberPausedEvent is triggered when a queue member is paused or unpaused.<br/>
 	/// It is implemented in apps/app_queue.c.<br/>
 	/// Available since Asterisk 1.2
 	/// </summary>
+    [Obsolete("Removed since Asterisk 13", false)]

```

Comment:
I don't think it should be a compile warning. it would confuse someone that's not using 12 upwards (11 is still supported for example from digium).